lib/rubocop/cop/util.rb

```diff
@@ -309,8 +309,8 @@ def compatible_external_encoding_for?(src)
         src.force_encoding(Encoding.default_external).valid_encoding?
       end
 
-      def line_distance(n1, n2)
-        n2.loc.line - n1.loc.line
```

Comment:
Same here with all the examples in this file. Rename ALL of them to something that embodies either a `AST::Node` or a `Parser::Source::Comment`.